### PR TITLE
adding sorting to roles list command

### PR DESCRIPTION
**Pull request by t0shiii**

### CODE REVIEW

File: commands/roles.js [hunk 1]
```diff
@@ -117,7 +117,7 @@ reason: 'dynamic role' });
     }
     case "list": {
       let allowedRoles = getAllowedRoles(client, channel.guild.id);
-      return channel.send(`${"Available roles:\n • "}${allowedRoles.join("\n • ")}`, {code: "asciidoc"});
+      return channel.send(`${"Available roles:\n • "}${allowedRoles.sort().join("\n • ")}`, {code: "asciidoc"});
```
[the remark this answers is not in the document]
Comment (t0shiii) [reply commands/roles.js]: done but please see other comment

File: commands/roles.js [hunk 2]
```diff
@@ -151,6 +151,7 @@ function registerNewRole(client, guildId, newRoleName) {
   const roleData = client.rolesData.ensure(guildId, []);
   roleData.push(newRoleName);
   client.rolesData.set(guildId, roleData);
+  client.rolesData = client.rolesData.sort();
```
Comment (t0shiii) [inline commands/roles.js:154]: per this link: https://www.npmjs.com/package/enmap/v/0.3.0#Enmap+sort

`The sort is not necessarily stable. `

Comment (mynameisivan) [reply commands/roles.js:154]: @I-AM-STEVEe and @BrandonWLee had test discords but I forget how they ran the bot - worst case we just test in production :)

**Review by mynameisivan — APPROVED** [review]

File: commands/roles.js (hunk 2 above)
Comment (I-AM-STEVEe) [reply commands/roles.js:154]: I can spin up mine if you need it.  It takes ~10 mins if you want to set up your own server too

**Pull request merged**